packages/validators-util/documentation/validators.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: Validators
+react: validators-util
+---
+
+import IsEmailExample from "./IsEmailExample";
+import IsPhoneNrExample from "./IsPhoneNrExample";
+
+<p>
+    <span class="jkl-lead">Denne pakken innholder validering støttefunksjoner.</span>
+</p>
```

Comment:
```suggestion
<p class="jkl-lead">Denne pakken innholder støttefunksjoner for validering av skjemaer.</span></p>
```

Reply:
```suggestion
<p class="jkl-lead">
    Denne pakken innholder validering støttefunksjoner.
</p>
```

isåfall.

Reply:
Jeg tenkte ikke over at dette er i `.mdx`-fila. Vi har en egen komponent for ingressen på disse sidene:

```
<Ingress>
    Denne pakken innholder støttefunksjoner for validering av skjemaer.
</Ingress>
```
Trenger ikke importeres noe for å bruke den.